qtpy/QtWidgets.py

```diff
@@ -7,10 +7,13 @@
 # -----------------------------------------------------------------------------
 
 """Provides widget classes and functions."""
-from functools import wraps
+from functools import partialmethod, wraps
+
+from packaging.version import parse
 
 from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, QtModuleNotInstalledError
-from ._utils import possibly_static_exec, getattr_missing_optional_dep
+from ._utils import add_action, possibly_static_exec, getattr_missing_optional_dep
+from .QtCore import __version__ as _qt_version
```

Comment:
Does `QT_VERSION` get exported that way? I'd prefer not to make the variable available from `QtWidgets.py`. That's why I rename it.

Secondly, as far as I can test, the `__version__` is equal to `QT_VERSION` in all the flavors for Qt6. That is, the binding version is the same as the one of Qt. Please check me here. If so, there is no difference. But will that go on like that? I haven't heard about such an agreement. 

Tomorrow, I'm back home from a 4-week vacation, and I'll re-test the Qt6 versions more consistently. And that's why it's completely OK for me that the review took a month.

Reply:
Oh I see, hope you had a good vacation !

Regarding the suggestion, `QT_VERSION` is defined here in the `__init__`:

 https://github.com/spyder-ide/qtpy/blob/9d1f3edd67f405d42a8468f4e50e9483a2352f0d/qtpy/__init__.py#L168-L173

Along side `PYQT5`, `PYQT6`, `PYSIDE2` and `PYSIDE6` and its value is initialized here for the different bindings:

https://github.com/spyder-ide/qtpy/blob/9d1f3edd67f405d42a8468f4e50e9483a2352f0d/qtpy/__init__.py#L189
https://github.com/spyder-ide/qtpy/blob/9d1f3edd67f405d42a8468f4e50e9483a2352f0d/qtpy/__init__.py#L217
https://github.com/spyder-ide/qtpy/blob/9d1f3edd67f405d42a8468f4e50e9483a2352f0d/qtpy/__init__.py#L240
https://github.com/spyder-ide/qtpy/blob/9d1f3edd67f405d42a8468f4e50e9483a2352f0d/qtpy/__init__.py#L253

So that's why I though that maybe we should use here the already defined constant. However, maybe getting some more feedback from others could help us define if the change is worthy :)

What do you think @ccordoba12 @CAM-Gerlach ?

Reply:
Done. Thank you for catching that!